examples/twittermap/noah/src/main/scala/edu/uci/ics/cloudberry/noah/adm/ADM.java

```diff
@@ -17,30 +15,46 @@
     public static final SimpleDateFormat ADMTimeFormat = new SimpleDateFormat("HH:mm:ss.SSS");
 
     public static String mkADMConstructor(String constructor, String content) {
-        return constructor + "(\"" + content + "\")";
+        StringBuilder sbConstructor=new StringBuilder();
```

Comment:
StringBuilder sbConstructor = new StringBuilder();

// Can you apply the correct source format? There was no space in the line.

Reply:
And why this string builder is necessary?